marimo/_server/api/endpoints/ws.py

```diff
@@ -257,6 +257,8 @@ async def get_session() -> Session:
             existing_session = mgr.get_session(session_id)
             if existing_session is not None:
                 LOGGER.debug("Reconnecting session %s", session_id)
+                # In case their is a lingering connection, close it
+                existing_session.maybe_disconnect_consumer()
```

Comment:
Just checking, does this interfere with waking from sleep?

Reply:
this actually fixes a case where the client did go to sleep but the backend did not disconnect (if through a proxy)